client-src/elements/chromedash-form-field.ts

```diff
@@ -57,9 +58,8 @@ export class ChromedashFormField extends LitElement {
   connectedCallback() {
     super.connectedCallback();
     this.fieldProps = ALL_FIELDS[this.name] || {};
-
     // Register this form field component with the page component.
-    const app = document.querySelector('chromedash-app');
+    const app: ChromedashApp | null = document.querySelector('chromedash-app');
```

Comment:
There must be a chromedash-app, so it would be reasonable to create a utility for getting this element, and throwing an error if it does not exist.

Reply:
this is a good point. added a utility function to get the element, but it still doesn't pass the webtest with the same error `      AssertionError: expected '\n      <!---->\n      <!--?lit$227800399$-->\n            <tr class="">\n              <th colspan="2">\n                <b><!--?lit$227800399$-->Unlisted:</b>\n              </th>\n            </tr>\n          \n      <tr class="">\n        <td><!--?lit$227800399$-->\n        <sl-checkbox size="small" name="unlisted" id="id_unlisted">\n          <!--?lit$227800399$-->A specific label\n        </sl-checkbox>\n       <!--?lit$227800399$--></td>\n        <td>\n          <!--?lit$227800399$--><span class="helptext"> <!--?lit$227800399$--> Check this box to hide draft features in list views. Anyone\n    with a link will be able to view the feature\'s detail page. </span>\n          <!--?lit$227800399$-->\n        </td>\n      </tr>\n\n      <!--?lit$227800399$-->\n    ' to include 'checked'`